client/js/components/procedure/admin/DpAddOrganisationList.vue

```diff
@@ -16,11 +16,85 @@
       class="u-mt-0_5"
       :header-fields="headerFields"
       :table-items="rowItems"
+      is-expandable
       is-selectable
       :items-per-page-options="itemsPerPageOptions"
       :default-sort-order="sortOrder"
       @items-selected="setSelectedItems"
       :init-items-per-page="itemsPerPage">
+      <template v-slot:expandedContent="{ participationFeedbackEmailAddress, locationContacts, ccEmailAddresses, contactPerson }">
+        <div class="lg:w-2/3 lg:flex pt-4">
+          <dl class="layout__item">
+            <dt class="color--grey">
+              {{ Translator.trans('address') }}
+            </dt>
+            <dd
+              v-if="locationContacts.street"
+              class="ml-0">
+              {{ locationContacts.street }}
+            </dd>
+            <dd
+              v-if="locationContacts.postalCode"
+              class="ml-0">
+              {{ locationContacts.postalCode }}
+            </dd>
+            <dd
+              v-if="locationContacts.city"
+              class="ml-0">
+              {{ locationContacts.city }}
+            </dd>
+            <dd
+              v-if="!locationContacts.street && !locationContacts.city && !locationContacts.postalCode"
+              class="ml-0">
+              {{ Translator.trans('notspecified') }}
+            </dd>
+          </dl>
+          <dl class="layout__item">
```

Comment:
Why do you use `layout__item` here?

Reply:
use tailwind classes instead here https://github.com/demos-europe/demosplan-core/pull/2630/commits/cb457846ceee564184e8828a383186c9713faac1